components/_footer.scss

```diff
@@ -12,3 +19,34 @@ footer {
     float: none;
   }
 }
+
+.ethos-rant {
+  color: $tws-white;
+
+  @include media($tablet) {
+    @include span-columns(5 of 10);
+  }
+}
+
+.admin-links {
+  font-family: $gibson;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets